server/db.mjs

```diff
@@ -6,13 +6,17 @@ const db = initDb();
 export const getTasks = async () => await db.any("SELECT * FROM tasks");
 //week get dates[0] dates[6] and query the range
 
-export const addTask = async (name) =>
+export const addTask = async (task) => {
+  const today = new Date();
+  const user_id = 1;
+
   (
-    await db.any("INSERT INTO tasks(name) VALUES($1) RETURNING id, name", [
-      name,
-    ])
+    await db.any(
+      "INSERT INTO tasks(task,date,user_id) VALUES($1,$2,$3) RETURNING id, task, date, user_id",
+      [task, today, user_id],
+    )
```

Comment:
This is failing for me because I don't have user 1 in my database -- where is this written? You may want to add an insert statement into your db seed file OR you can do a check when your app starts and if there are no users, insert one row (for now - for testing purposes)